StepsTracker.empty-codes/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "StepsTracker.empty-codes": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\Morayo\\Documents\\GitHub\\CodeReviews.Console.HabitTracker\\StepsTracker.empty-codes\\"
```

Comment:
Note that when you include hardcoded paths like this you need to let others know that they need to update them in order to run your app!

Reply:
Thank you for the valuable feedback! I have taken note of your suggestions and I will make sure to apply them going forward. Thank you once again 🙏